react/package.json

```diff
@@ -22,7 +22,7 @@
     "tidy-up": "rimraf ./R && rimraf ./dist && rimraf ./inst && rimraf ./man && rimraf ./.Rbuildignore",
     "build:js": "webpack --mode production --entry ./dist/index.js",
     "build:js-dev": "webpack --mode development --entry ./dist/index.js",
-    "build:py_and_r": "npm run transpile:dash && cp -a ../webviz_core_components/. ./webviz_core_components/ && cp ./package.json ./webviz_core_components/package.json && dash-generate-components ./dist/components webviz_core_components -p package.json --r-prefix '' --ignore '(^index.js|.stories.js)$' && cp -a ./webviz_core_components/. ../webviz_core_components/ && rimraf ./webviz_core_components",
+    "build:py_and_r": "npm run transpile:dash && cp -aR ../webviz_core_components/. ./webviz_core_components/ && cp ./package.json ./webviz_core_components/package.json && dash-generate-components ./dist/components webviz_core_components -p package.json --r-prefix '' --ignore '(^index.js|.stories.js)$' && cp -a ./webviz_core_components/. ../webviz_core_components/ && rimraf ./webviz_core_components",
```

Comment:
Just for understanding, what does adding `-a` solve?

Reply:
Since I added the `a` :wink:
`-a` is a shortcut for `-dR --preserve=all`. 
- `d`: shortcut for `--no-dereference --preserve=links` (`never follow symbolic links in SOURCE`, preserve links)
- `R`: copy directories recursively
- `--preserve=all`: preserves all attributes (mode, ownership, timestamps..)

Hence, I am a bit surprised that you needed to add the `R` since it should be included in the `-a` flag.

Reply:
I you are right that `R` is not needed. Will remove!